src/Services/Terminal/Terminal.js

```diff
@@ -67,6 +73,57 @@ process.on('exit', () => {
   });
 });
 
+const getErrorTagPrefix = (detailObject) => {
+  const [field] = Object.keys(detailObject);
+  const [prefix] = field.split(':');
+
+  return prefix;
+};
+
+const isErrorRetriable = (e) => {
+  if (!e.causedBy || !e.causedBy.data || !e.causedBy.data.detail) {
+    return false;
+  }
+
+  const { detail } = e.causedBy.data;
+  const versionPrefix = getErrorTagPrefix(detail);
+  const errorCode = detail[`${versionPrefix}:${ERROR_INFO_SUFFIX}`]
+    && detail[`${versionPrefix}:${ERROR_INFO_SUFFIX}`][`${versionPrefix}:${ERROR_CODE_SUFFIX}`];
+  if (errorCode && RETRY_CODES_LIST.includes(errorCode)) {
+    return true;
+  }
+
+  return false;
+};
+
+const commandRetry = async ({
+  service,
+  command,
+  sessionToken,
+  times = DEFAULT_RETRY_TIMES,
+  count = 0
```

Comment:
Unnecessary use of `count`. You may use times, gradually decreasing and throw an error when the `times = 0`